### ISSUE TITLE
Ensure API Readme is understandable to non-technical staff 

### ISSUE THREAD

**Issue by GovInTrenches** [hurricane-response/florence-api#47]
We should review the documentation to make sure that if we're making it readable for non-technical staff. Specifically, if a member of FEMA or other disaster organization reads this, it should be clear what this actually does in plain English

**Comment by Kaushl2208** [comment]
Can i work on this?? @GovInTrenches

**Comment by miklb** [comment]
Yes absolutely. We would love your help. Let us know if you have any questions.
